editor/inserter/menu.js

```diff
@@ -305,6 +313,9 @@ export class InserterMenu extends Component {
 	}
 
 	switchTab( tab ) {
+		// store the scrollTop of the tab switched from
+		this.tabScrollTop[ this.state.tab ] = this._tabContainer.scrollTop;
+		this.didSwitchTab = true;
```

Comment:
I don't think we need this variable. Instead, `componentDidUpdate` receives the previous state as an argument, so we can check whether tab changed:

```js
componentDidUpdate( prevProps, prevState ) {
	// ...

	const didSwitchTab = this.state.tab !== prevState.tab;
	if ( didSwitchTab ) {
		// ...
```